### ISSUE TITLE
Additional parameter to default sideBar start closed

### ISSUE THREAD

**Issue by johnkangw** [plotly/dash-ag-grid#25]
Enabling the sideBar (sideBar = True) in the dag.AgGrid component defaults the sidebar open in the grid. It would be great to have a parameter that I could set to default the sidebar to close.

**Comment by BSd3v** [comment]
Hello @johnkangw,

For configuration options, we left these as wide as ag-grid themselves would allow.

If you take a look here: https://www.ag-grid.com/react-data-grid/side-bar/#sidebardef-configuration

You can see all the configuration options, to start closed, you will need to pass `"defaultToolPanel": None` to the configuration.

Here is an example:
```
sideBar={"toolPanels": [
        {
            "id": 'columns',
            "labelDefault": 'Columns',
            "labelKey": 'columns',
            "iconKey": 'columns',
            "toolPanel": 'agColumnsToolPanel',
            "minWidth": 225,
            "maxWidth": 225,
            "width": 225
        },
        {
            "id": 'filters',
            "labelDefault": 'Filters',
            "labelKey": 'filters',
            "iconKey": 'filter',
            "toolPanel": 'agFiltersToolPanel',
            "minWidth": 180,
            "maxWidth": 400,
            "width": 250
        }
    ],
    "position": 'right',
    "defaultToolPanel": None}
```

**Comment by AnnMarieW** [comment]
Great!  Let's leave this issue open until we can add it as an example in the docs.

**Comment by johnkangw** [comment]
@AnnMarieW and @BSd3v Thanks so much! I defined the sideBar and got it working. Random question, but I see in the "filters 2" for more filters. What is that used for?

**Comment by BSd3v** [comment]
It’s just to allow you to create new tabs in the event that you want to create a different way to operate the filters on the grid.

**Comment by AnnMarieW** [comment]
Closed by #103
